plugin/listtoggle.vim

```diff
@@ -18,11 +18,6 @@
 "
 " See ':help listtoggle' for more information.
 
-if exists('g:loaded_listtoggle')
-  finish
-endif
-let g:loaded_listtoggle = 1
```

Comment:
I'm sorry having not realized it. It's my first time trying to contribute to a vim plugin. These lines should be a common practice. I will correct it latter.